styless.css

```diff
@@ -0,0 +1,155 @@
+body {
+  font-family: 'Old Standard TT', serif;
+  line-height: 1.4;
+  margin: 0;
+  background: rgba(0, 214, 170, .9);
+  font-weight: 400;
+}
+
+.container {
+  max-width: 960px;
+  margin: auto;
+  overflow: hidden;
+  padding: 0 30px;
+}
+
+.showcase {
+  background-color: rgba(130, 149, 166, .75);
+  color: #fff;
+  height: 100vh;
+  position: relative;
+}
+
+.showcase::before {
+  content: '';
+  background: url(wallpaper.jpg) no-repeat center center/cover;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.showcase .showcase-inner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  height: 100%;
+}
+
+.btn {
+  display: inline-block;
+  border: none;
+  background-color: rgba(130, 149, 166, .75);
+  color: white;
+  padding: 12px 24px;
+  margin-top: 16px;
+  text-decoration: none;
+  transition: opacity 1s ease-in;
+}
+
+@media (max-width: 800px) {
+  nav {
+    transform: scale(1, 0);
+  }
+
+  nav a {
+    opacity: 0;
+  }
+
+  .toggler {
+    position: absolute;
+    z-index: 2;
+    cursor: pointer;
+    width: 30px;
+    height: 30px;
+    display: none;
+  }
+
+  .toggler:checked ~ nav {
+    display: block;
+    transform: scale(1, 1);
+  }
+
+  .toggler-label {
+    position: absolute;
+    top: 0;
+    left: 0;
+    margin-left: 16px;
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  .toggler-label span,
+  .toggler-label span::before,
+  .toggler-label span::after {
+    display: block;
+    background: white;
+    height: 3px;
+    width: 32px;
+    position: relative;
+  }
+
+  .toggler-label span::before,
+  .toggler-label span::after {
+    content: '';
+    position: absolute;
+  }
+
+  .toggler-label span::before {
+    bottom: 8px;
+  }
+
+  .toggler-label span::after {
+    top: 8px;
+  }
+
+  .toggler:checked ~ nav a {
+    opacity: 1;
+    transition: opacity .3s ease-in-out .3s;
+  }
+}
+
+@media (min-width: 400px)
+  and (max-width: 600px) {
+  h1 {
+    font-size: 28px;
+  }
+
+  p,
+  a {
+    font-size: 16px;
+  }
+}
+
+@media (min-width: 601px)
+  and (max-width: 800px) {
+  h1 {
+    font-size: 42px;
+  }
+
+  p,
+  a {
+    font-size: 22px;
+  }
+}
+
+@media (min-width: 801px) {
+  h1 {
+    font-size: 48px;
+  }
+
+  p,
+  a {
+    font-size: 28px;
+  }
+}
+
+.btn:hover {
+  opacity: .7;
+}
+   
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/cheeseburger/pullRequest?prid=6066612)